modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -178,18 +190,19 @@ class ImagingUploader extends Component {
       }
     }
 
-    if (column === 'Number Of Files Created') {
+    if (column === filesCreatedKey) {
       let violatedScans;
       if (
-        row['Number Of Files Created'] - row['Number Of Files Inserted'] > 0
+        row[filesCreatedKey] - row[filesInsertedKey] > 0
       ) {
         let numViolatedScans =
-             row['Number Of Files Created'] - row['Number Of Files Inserted'];
+             row[filesCreatedKey] - row[filesInsertedKey];
 
         const violUrl = loris.BaseURL +
                          '/mri_violations/?patientName=' + row.PatientName;
         violatedScans = <a href={violUrl}>
-           ({numViolatedScans} violated scans)
+          {numViolatedScans} + {' '}
+          {this.props.t('violated scans', {ns: 'imaging_uploader'})}
```

Comment:
I think the {numViolatedScans} part probably needs to be inside the t() interpolation, not in react interpolation?